pyvista/plotting/plotting.py

```diff
@@ -1293,8 +1310,8 @@ def prepare_mapper(scalars):
             prop.SetLineWidth(line_width)
 
         # Add scalar bar if available
-        if stitle is not None and show_scalar_bar and (not rgb or _custom_opac):
-            self.add_scalar_bar(stitle, **scalar_bar_args)
+        if title is not None and show_scalar_bar and (not color or _custom_opac):
+            self.add_scalar_bar(title, **scalar_bar_args)
```

Comment:
This change is incorrect and not needed for the desired results. This change adds a scalar bar when one is not supped to be there. Also, the docs are failing for me locally (I'm not sure why Travis didn't pick that up 😨 )

```py
from pyvista import examples
examples.load_airplane().plot()
```

![download](https://user-images.githubusercontent.com/22067021/73333259-78bc0580-4226-11ea-973e-9b32f3a6e12a.png)

Reply:
hmmm it seems that any of the versions of my edits and the original behavior all have some issue.  Should all the edits here be reverted?  The naming of the vector norm and components will not work, but that is the current state.

Reply:
I'm now thinking that `title` was correct here and the `stitle` in the vector valued portion should be changed to `title`.  But, every other thought I've had was wrong.  I'm going to suggest that all of the scalar bar title commits be reverted back to the original state.  Then I'll open a new issue to track this issue.